doc/styles/config/vocabularies/ANSYS/accept.txt

```diff
@@ -9,8 +9,14 @@ link_code_library
 link_code_source
 link_code_branch
 html_context
-autoapi
+(?i)autoapi
```

Comment:
Just curious about what this syntax means.

Reply:
This is related to case-insensitivity, i.e. the matching should be case-insensitive.